libs/api/domains/syslumenn/src/lib/client/models/dataUpload.ts

```diff
@@ -45,6 +46,7 @@ export function constructUploadDataObject(
   id: string,
   persons: Person[],
   attachment: Attachment,
+  extraData: object,
```

Comment:
Should this maybe be an optional argument?

Reply:
A more specific type would also be nice if that's possible :)

Reply:
1. Let's change it to an optional argument if someone uses this endpoint in the feature without using the extra data. Since all callers are using extra data now then I would like to have it as mandatory. 
2. Fixed 😄